app/assets/stylesheets/static_pages_home.scss

```diff
@@ -0,0 +1,41 @@
+.static_pages_home {
+	height: 100vh; width: 100vw;
+	background: #F0F0F0;
+
+	.row {
+		position: relative; top: 45%;
```

Comment:
This should be 50%.
Probably 50% doesn;t seem to center perfectly due to unnecessary margin for one of the elements.
Either remove the margin or add a transparent border around it.